test/integration/blocks-schema.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import Ajv from 'ajv-draft-04';
```

Comment:
I had to install `ajv-draft-04` to make everything work with the defined schema: 
http://json-schema.org/draft-04/schema#

> Out of the box, Ajv supports JSON Schema (drafts 04, 06, 07, 2019-09 and 2020-12) and JSON Type Definition (RFC8927 (opens new window)).

However, in practice, only JSON Schema draft-06/07/2019-09/2020-12 standards work with the core `ajv` package. Is there any reason we picked `draft-04`?

Reply:
@gziolo This is all the context I have. We briefly talked about it in the initial review of the `block.json` introduction PR. 

https://github.com/SchemaStore/schemastore/pull/1879#issuecomment-944426503

Reply:
I guess it’s fine to leave it as is. Thank you for sharing the context 💯

Reply:
SchemaStore has some best practices listed in their contributing docs that I was also basing my decisions off of. https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices